PocketKit/Sources/PocketKit/LoggedOut/LoggedOutViewModel.swift

```diff
@@ -173,11 +173,13 @@ class LoggedOutViewModel: ObservableObject {
                     // but the other errors should never occur, so they should be captured.
                     switch nested.code {
                     case .presentationContextInvalid, .presentationContextNotProvided:
+                        Crashlogger.breadcrumb(category: "auth", level: .error, message: "ASWebAuthenticationSessionError: \(nested.localizedDescription)")
```

Comment:
I think the Sentry bug error is due to this location and not the other areas, but added breadcrumbs in the other sections since it can't hurt to have more data. We can always remove or modify in the future.

Reply:
@cyndichin Have you "manually" verified that the localized description is something more human readable? Just want to double check that it's not going to spit out something "useless".

Reply:
This is what gets outputted in the playground, let me know if you think another message that is more useful.
![image](https://user-images.githubusercontent.com/6743397/208149796-33b1be82-5ff3-4fbb-a385-6369cb5c8168.png)

Reply:
![image](https://user-images.githubusercontent.com/6743397/208149886-d04686cf-dfa4-41b4-b26d-23b43bf4287b.png)

Reply:
I think this is valuable enough.